Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -46,14 +46,31 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         },
 
         setUp: function () {
-            this.model = new Y.eZ.Location();
+            this.locationId = '1/2/3';
+            this.model = new Y.eZ.Location({id: this.locationId});
+            this.contentServiceMock = new Y.Mock();
         },
 
         tearDown: function () {
             this.model.destroy();
             delete this.model;
-        }
+        },
+
+        "Should have a move method": function () {
+            var callback = function () {},
+                parentLocationId = '4/5/6';
 
+            Y.Mock.expect(this.capiMock, {
+                method: 'getContentService',
+                returns: this.contentServiceMock
+            });
+            this.options = {api: this.capiMock};
```

Comment:
why putting the options as a property of the test case ? A local variable would be better suited.